requirements.txt

```diff
@@ -0,0 +1,3 @@
+jplephem>=2.3
```

Comment:
I would prefer to have only one file be in charge of what gets installed for developers, and at the moment that's `requirements.sh`. I'm not opposed to discussing whether it could, given how much healthier the Python ecosystem is these days, be replaced with a `requirements.txt`, but:

1. The file would need to do everything `requirements.sh` does.
2. And, the file should not duplicate information already in the `setup.py`. It should ask for this package "skyfield" to be installed, which will add automatically the dependencies in `setup.py`.

Reply:
I'm not used to looking for `requirements.sh`, just `requirements.txt`. Possibly moving the dependencies to `requirements.txt` and then `requirements.sh` can read/use it, as well as `pip3 install -r requirements.txt`

Reply:
I'd be happy with a pull request that converted `requirements.sh` to `requirements.txt` — I have gradually stopped using conda for everything, now that wheels have become common enough.